### ISSUE TITLE
Upgrade from ??? to latest docker image fails "entrypoint.sh: line 89: gosu: command not found"

### ISSUE THREAD

**Issue by timdonovanuk** [photoprism/photoprism#2149]
Hi,

I just tried recreating my docker container with the latest image (photoprism/photoprism:latest) and the container now fails to start:

```
2022-03-14T10:28:16.962940569Z init as root
2022-03-14T10:28:16.969767258Z usermod: no changes
2022-03-14T10:28:16.969987462Z image: amd64-prod, build 220302-bullseye
2022-03-14T10:28:16.970010331Z home: /photoprism
2022-03-14T10:28:16.970840856Z umask: "0002" (u=rwx,g=rwx,o=rx)
2022-03-14T10:28:16.970870798Z install-path: /opt/photoprism
2022-03-14T10:28:16.971627879Z switching to uid 1000:1000
2022-03-14T10:28:16.971652322Z photoprism start
2022-03-14T10:28:16.972199515Z /entrypoint.sh: line 89: gosu: command not found
```

I'm not sure what version I was trying to upgrade from, as there was no easily identifiable version number in Photoprism. I took this screenshot, as the text isn't selectable:

![2022-03-14 10_23_43-PhotoPrism_ Browse Your Life](https://user-images.githubusercontent.com/8156439/158154009-97bddb9c-c9d3-4728-8559-8e2e92968507.png)

**Comment by graciousgrey** [comment]
This sounds similar to: https://github.com/photoprism/photoprism/discussions/2120

**Comment by timdonovanuk** [comment]
Happy to share my compose file:
```

version: '3.9'

x-logging:
  &loki-logging
  driver: syslog
  options:
    syslog-address: "udp://192.168.1.52:514"
    syslog-format: "rfc5424micro"
    tag: "{{.Name}}"


services:

    photoprism:
        container_name: photoprism
        image: 'photoprism/photoprism:latest'
        restart: unless-stopped
        logging: *loki-logging
        security_opt:
          - 'seccomp:unconfined'
          - 'apparmor:unconfined'
        ports:
          - '2342:2342'
        environment:
          PHOTOPRISM_ADMIN_PASSWORD: *****
          PHOTOPRISM_HTTP_PORT: 2342
          PHOTOPRISM_HTTP_COMPRESSION: gzip
          PHOTOPRISM_DEBUG: 'false'
          PHOTOPRISM_PUBLIC: 'true'
          PHOTOPRISM_READONLY: 'false'
          PHOTOPRISM_EXPERIMENTAL: 'false'
          PHOTOPRISM_DISABLE_WEBDAV: 'false'
          PHOTOPRISM_DISABLE_SETTINGS: 'false'
          PHOTOPRISM_DISABLE_TENSORFLOW: 'false'
          PHOTOPRISM_DARKTABLE_PRESETS: 'false'
          PHOTOPRISM_DETECT_NSFW: 'true'
          PHOTOPRISM_UPLOAD_NSFW: 'true'
          PHOTOPRISM_DATABASE_DRIVER: mysql
          PHOTOPRISM_DATABASE_SERVER: '192.168.1.6:3306'
          PHOTOPRISM_DATABASE_NAME: photoprism
          PHOTOPRISM_DATABASE_USER: photoprism
          PHOTOPRISM_DATABASE_PASSWORD: ******
          PHOTOPRISM_SITE_URL: 'http://localhost:2342/'
          PHOTOPRISM_SITE_TITLE: PhotoPrism
          PHOTOPRISM_SITE_CAPTION: Browse Your Life
          PHOTOPRISM_SITE_DESCRIPTION: ''
          PHOTOPRISM_SITE_AUTHOR: ''
          PUID: 1000
          PGID: 1000
        volumes:
          - '/photos/originals:/photoprism/originals'
          - '/photos/import:/photoprism/import'
          - '/photos/photoprism:/photoprism/storage'
```

**Comment by timdonovanuk** [comment]
Ah sorry just saw the the answer in the other thread, let me check and confirm.

**Comment by timdonovanuk** [comment]
Getting one step closer, now it's throwing a different error:

```
init as root
usermod: no changes
image: amd64-prod, build 220302-bullseye
home: /photoprism
umask: "0002" (u=rwx,g=rwx,o=rx)
install-path: /opt/photoprism
switching to uid 1000:1000
photoprism start
time="2022-03-14T10:38:06Z" level=info msg="PhotoPrism® needs your support!"
time="2022-03-14T10:38:06Z" level=info msg="Visit https://docs.photoprism.app/funding/ to learn more."
time="2022-03-14T10:38:12Z" level=info msg="migrate: found no previous migrations"
time="2022-03-14T10:38:12Z" level=info msg="migrate: 20211121-094727 successful [9.531191ms]"
time="2022-03-14T10:38:12Z" level=info msg="migrate: 20211124-120008 successful [5.419329ms]"
time="2022-03-14T10:38:12Z" level=info msg="migrate: 20220103-115400 successful [10.329477ms]"
time="2022-03-14T10:38:12Z" level=info msg="migrate: 20220118-172400 successful [82.816239ms]"
time="2022-03-14T10:38:12Z" level=info msg="http: enabling gzip compression"
time="2022-03-14T10:38:12Z" level=error msg="html/template: pattern matches no files: `/photoprism/assets/templates/*`"
```

**Comment by timdonovanuk** [comment]
Cleared PATH and PHOTOPRISM_ASSETS_PATH env variables and all good now, thanks!

**Comment by lastzero** [comment]
Re-added HOME to our https://dl.photoprism.app/docker/docker-compose.yml example and updated our scripts to use absolute command paths, so changing PATH should not break the image anymore.

**Comment by fhsdjk** [comment]
Hi,
I'm using a Synology DiskStation with Docker with no further applications or configuration. 
I deployed the image with version PhotoPrism® 220121-2b4c8e1f-Linux-x86_64  from the registry in ~February.
Stopping & renaming the image, duplicating the settings and starting it with the latest image from the registry (yesterday) fails for me with the same error as above:
/entrypoint.sh: line 89: gosu: command not found

I removed the PATH variable completely from the settings, also the PHOTOPRISM_xxxxx_PATH ones and added the HOME=/photoprism in various combinations, but got no further luck.
Any suggestions?
